index.js

```diff
@@ -92,8 +122,14 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+
+//   let newGenre = movies.filter(
+//     (movie) => movie.genre === genre
+//   );
+// return newGenre
```

Comment:
You were really close on this one too! movie.genre is actually a string of _several_ genres, not just one. That's why === won't work here and you have to use .includes instead. You also aren't using .toLowerCase to make it case insensitive, but i didn't include that in my suggested code here because I want to emphasize using .includes vs. === . 
```suggestion
let newGenre = movies.filter(
 (movie) => movie.genre.includes(genre)
 );
return newGenre
```
We can go over this 1:1 during class or office hours too.